homework/hackyourtemperature/routes/weather.js

```diff
@@ -0,0 +1,24 @@
+import express from "express";
+import weatherData from "../data.js";
+import fetchAPI from "../helpers/fetch.js";
+
+const router = express.Router();
+
+router.get("", (req, res) => {
+  res.render("index", { text: "" });
+});
+
+router.post("/weather", async (req, res) => {
+  const cityName = req.body.cityName;
+  const jsonData = await fetchAPI(cityName);
+  if (jsonData.cod === 200) {
```

Comment:
I think this should probably be `code`. 😆 It's important to test these conditional statements in your unit tests because it wouldn't have worked!

```suggestion
  if (jsonData.code === 200) {
```